rfc6962-bis.xml

```diff
@@ -658,8 +658,7 @@ to verify the SCT signature.
 
      struct {
          SerializedSCT sct_list &lt;1..2^16-1&gt;;
-     } SignedCertificateTimestampList;
-          </artwork>
+     } SignedCertificateTimestampList;</artwork>
```

Comment:
I deliberately moved the closing artwork tag to the end of the previous line, to get rid of a blank empty line at the bottom of the artwork (in the HTML output from http://xml2rfc.ietf.org/ at least).